Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11155.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.Frame)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 11155,
+		"Picking option in picker crashes application - iOS ONLY",
+		PlatformAffected.iOS)]
+	public class Issue11155 : TestContentPage
+	{
+		public Issue11155()
+		{
+		}
+
+		protected override void Init()
+		{
+			Title = "Issue 11155";
+
+			var layout = new StackLayout
+			{
+				Padding = 12
+			};
+
+			var instructions = new Label
+			{
+				Text = "Select any Picker item."
```

Comment:
Just selecting a Picker item doesn't trigger the bug - the user has to use the Done button.

Reply:
Updated the instructions.